lib/startErrorHandling.js

```diff
@@ -26,13 +26,28 @@ var startErrorHandling = function startErrorHandling(store) {
 
   raven.config(_constants.CONFIG.ravenURL).install();
 
+  var checkUUID = function checkUUID(_ref, _ref2) {
+    var prevWorker = _ref.worker;
+    var curWorker = _ref2.worker;
+
+    if (!prevWorker.get('uuid') && curWorker.get('uuid')) {
```

Comment:
yeah but we can't always know it because the worker might not be logged in (the login comes through a message from the tester profile to the chrome extension). The whole app is designed to handle null UUID and start things up once it's set.